packages/theme/api-report.md

```diff
@@ -70,6 +70,7 @@ export type BackstagePaletteAdditions = {
     error: string;
     text: string;
     link: string;
+    closeButtonColor: string;
```

Comment:
Wonder if we should make this optional and have a sensible fallback, that way we don't break typescript for everyone.

Reply:
Yeah, I guess that will work out better. I'll add it as an additional commit to get your feedback on it :)